lib/stdlib/doc/src/gen_event.xml

```diff
@@ -1005,6 +1015,12 @@ gen_event:stop             ----->  Module:terminate/2
           an event manager receives any other message than an event or
           a synchronous request (or a system message).</p>
         <p><c>Info</c> is the received message.</p>
+        <p>This callback will receive <c>'EXIT'</c> messages from processes
+          which terminated after calling
+          <seemfa market="#add_sup_handler/3">
```

Comment:
using _marker_ attribute instead of _market_ will fix the docs build error.
you can check "make docs" command yourself before pushing

Reply:
Build seems broken for reasons I'm not clear on (I just did `./configure && make docs`), so I'll let the Github CI tell me off:

```
 ERLC   ../ebin/edoc_layout_chunks.beam
edoc_layout_chunks.erl:58: can't find include lib "kernel/include/eep48.hrl"
edoc_layout_chunks.erl:61: record docs_v1 undefined
edoc_layout_chunks.erl:458: record docs_v1 undefined
edoc_layout_chunks.erl:457: Warning: variable 'Anno' is unused
edoc_layout_chunks.erl:457: Warning: variable 'Docs' is unused
edoc_layout_chunks.erl:457: Warning: variable 'Metadata' is unused
edoc_layout_chunks.erl:457: Warning: variable 'ModuleDoc' is unused
```